libs/contrib/Data/String/Parser.idr

```diff
@@ -170,6 +203,11 @@ export
 spaces : Parser ()
 spaces = skip (many space) <?> "white space"
 
+||| Discards brackets around a matching parser
+export
+parens : Parser a -> Parser a
```

Comment:
Is there a reason all of these are suddenly about `Parser` rather than `ParserT`?

Reply:
Ah, that might be from when I added #526. Copied some parsers from my test into this file, and think I was using Parser rather than ParseT